guides/legacy/asset-compilation.md

```diff
@@ -0,0 +1,622 @@
+<!-- Unedited copy of asset-compilation -->
+
+### Raw Assets
+
+* `public/assets` vs `app/styles`
+
+To add images, fonts, or other assets, place them in the `public/assets` directory. For
+example, if you place `logo.png` in `public/assets/images`, you can reference it in
+templates with `assets/images/logo.png` or in stylesheets with
+`url('/assets/images/logo.png')`.
+
+This functionality of Ember CLI comes from
+[broccoli-asset-rev](https://github.com/rickharrison/broccoli-asset-rev). Be
+sure to check out all the options and usage notes.
+
+### JS Transpiling
+
+Ember CLI automatically transpiles future JavaScript (ES6/ES2015, ES2016 and beyond) into standard ES5
+JavaScript that runs on every browser using [Babel JS](https://babeljs.io) with the [Ember CLI Babel](https://github.com/babel/ember-cli-babel) addon.
+
+Internally, Ember CLI Babel uses `babel-preset-env`, which figures out which parts of your code
+need to be transpiled to ES5 by **analyzing your project's browser support targets**. A `target` is a special keyword
+that maps to a [browserlist](https://github.com/ai/browserslist) support rule. These are defined in your
+`config/targets.js` file, which [Ember CLI generates](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/config/targets.js) like so:
+
+```js
+/* eslint-env node */
+module.exports = {
```

Comment:
could we change this with the content from the latest target.js from the latest blueprint?